sealed-secrets.yaml

```diff
@@ -0,0 +1,111 @@
+package:
+  name: sealed-secrets
+  version: 0.28.0
+  epoch: 0
+  description: A Kubernetes controller and tool for one-way encrypted Secrets
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/bitnami-labs/sealed-secrets
+      tag: v${{package.version}}
+      expected-commit: 6b1b331a2cd3a58569ce4d819a7cabc59c0a3e50
+
+  - uses: go/build
+    with:
+      packages: ./cmd/controller
+      output: controller
+      ldflags: -X main.VERSION=${{package.version}}
+
+subpackages:
+  - name: ${{package.name}}-kubeseal
+    dependencies:
+      provides:
+        - kubeseal=${{package.full-version}}
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/kubeseal
+          output: kubeseal
+          ldflags: -X main.VERSION=${{package.version}}
+    test:
+      pipeline:
+        - runs: |
+            kubeseal --version | grep ${{package.version}}
+
+  - name: ${{package.name}}-kubeseal-bitnami-compat
+    description: "compat package with bitnami/sealed-secrets-kubeseal image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: sealed-secrets-kubeseal
+          version-path: 0/debian-12
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/sealed-secrets-kubeseal/bin/
+          # this is conflicting with ca-certificates-bundle
+          rm -f ${{targets.contextdir}}/etc/ssl/certs/ca-certificates.crt
+          ln -sf /usr/bin/kubeseal ${{targets.contextdir}}/opt/bitnami/sealed-secrets-kubeseal/bin/kubeseal
+          ln -sf /usr/bin/kubeseal ${{targets.contextdir}}/kubeseal
+    test:
+      pipeline:
+        - runs: |
+            stat /opt/bitnami/sealed-secrets-kubeseal/bin/kubeseal
+            stat /kubeseal
+
+  - name: ${{package.name}}-bitnami-compat
+    description: "compat package with bitnami/sealed-secrets-controller image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: sealed-secrets-controller
+          version-path: 0/debian-12
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/sealed-secrets-controller/bin/
+          # this is conflicting with ca-certificates-bundle
+          rm -f ${{targets.contextdir}}/etc/ssl/certs/ca-certificates.crt
```

Comment:
First time, I'm seeing this do we know that the files the one we add by ca-certificates-bundle package and ca-certificates.crt in this case? I just wanted to make sure that if the file ca-certificates.crt is something required for Bitnami version of sealed-secrets-controller

Reply:
thank you for looking into this, this is vendored by upstream here https://github.com/bitnami/containers/blob/main/bitnami/sealed-secrets-kubeseal/0/debian-12/rootfs/etc/ssl/certs/ca-certificates.crt 

our package provides the following:
```bash
$ apk info -L ca-certificates-bundle
ca-certificates-bundle-20241121-r2 contains:
etc/pki/tls/certs/ca-bundle.crt
etc/ssl/cert.pem
etc/ssl/certs/ca-bundle.crt
etc/ssl/certs/ca-certificates.crt
var/lib/db/sbom/ca-certificates-bundle-20241121-r2.spdx.json
```
removed the upstream once here so that we can use our package.

Reply:
+1, can we diff to compare with ours whether this is bitnami-specific thing? https://github.com/bitnami/containers/blob/main/bitnami/sealed-secrets-controller/0/debian-12/rootfs/etc/ssl/certs/ca-certificates.crt

Reply:
Then we might need to run `update-ca-certificates`